docs-src/python/how-to-set-an-activity-retry-policy-in-python.md

```diff
@@ -3,20 +3,26 @@ id: how-to-set-an-activity-retry-policy-in-python
 title: How to set an Activity Retry Policy in Python
 sidebar_label: Retry Policy
 description: Create an instance of an Activity Retry Policy in Python.
-tags:
-  - python
-  - how-to
 ---
 
 To create an Activity Retry Policy in Python, set the [RetryPolicy](https://python.temporal.io/temporalio.common.RetryPolicy.html) class within the [`start_activity()`](https://python.temporal.io/temporalio.workflow.html#start_activity) or [`execute_activity()`](https://python.temporal.io/temporalio.workflow.html#execute_activity) function.
 
-The following example sets the maximum interval to 2 seconds.
+<a class="dacx-source-link" href="https://github.com/temporalio/documentation-samples-python/blob/main/activity_timeouts_retires/your_workflows_dacx.py">View source code</a>
 
 ```python
-workflow.execute_activity(
-    your_activity,
-    name,
-    start_to_close_timeout=timedelta(seconds=10),
-    retry_policy=RetryPolicy(maximum_interval=timedelta(seconds=2)),
-)
+from temporalio.common import RetryPolicy
+# ...
+        your_retry_policy = await workflow.execute_activity(
```

Comment:
This is confusing to assign an activity result to a variable named like it's a retry policy

Reply:
But it's not a retry policy result, it's an activity result